amida-tech/amida-survey-microservice
# Feature/ser 18 list completed surveys

#### What's this PR do?
Added a parameter to the user-surveys endpoint called "user-survey-status' to allow for filtering by user status/progress. 
#### Related JIRA tickets:
[SER-17](https://jira.amida-tech.com/browse/SER-17)
[SER-18](https://jira.amida-tech.com/browse/SER-18)

#### How should this be manually tested?
Create multiple users, fill out several surveys under each user, and check that the endpoint correctly returns the surveys filtered by progress, new, complete or in-progress.

#### Any background context you want to provide?
Currently, when a survey is created it is available to all users. So all surveys that are not started by a user, but exist in the system, will be listed as 'new' under that user. 

we may need to change this functionality to assign specific surveys to specific users. 
#### Screenshots (if appropriate):
